snapcraft/internal/project_loader/_extensions/gnome_3_28.py

```diff
@@ -25,9 +25,28 @@
 
 
 class Gnome_3_28Extension(Extension):
-    """The Gnome extension.
+    """This extension eases creation of snaps that integrate with Gnome 3.28
```

Comment:
s/Gnome/GNOME?

Reply:
I will leave this one up to @kenvandine but if not comment from him, I do not mind changing

Reply:
GNOME is appropriate